doc/howto_release.md

```diff
@@ -263,8 +274,8 @@ you can get the same or similar message again using the script
 
 ## Publish release
 
-For the final release, edit the draft release again and publish it using the
-"Publish release" button.
+For the final release, edit the draft release again in order to publish it
+using the "Publish release" button.
```

Comment:
What's wrong with the original sentence?

Just for fun, I put both versions to DeepL Write. For the original sentence, it only modify the r to R in release and messes up quotes. For the new version it suggests:

> For the final release, edit the draft release again to publish it with the "Publish Release" button.

...which is wrong. What you are trying to say? Is there a UI glitch? An edit is necessary to have the publish button appear?

The reason why you need to edit the draft release again is that you should have better release notes from the RC1 which you now need to combine with the fixes for the final release if applicable. The auto-generated notes will be okay, but worse than what you already did for the RC1. ...at least I think that's the procedure. Maybe worth mentioning here.

Reply:
Wording refined.